include/alp/reference/matrix.hpp

```diff
@@ -565,7 +641,7 @@ namespace alp {
 				typedef std::function< Ret( Args... ) > lambda_function_type;
 				lambda_function_type &lambda;
 
-				T get( const size_t i, const size_t j ) const {
+				access_type access( const size_t i, const size_t j ) const {
```

Comment:
I think we should keep a coherent interface between derivations of MatrixBase. Should we have `access` obtain a single "storage index" (a pair in the case of functors) as it happens in the container-type Matrix?

Reply:
Fixed in the latest push